<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="welcomeLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 14pt; font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Welcome, User (Role)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="itemTab">
       <attribute name="title">
        <string>Available Items</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="itemButtonLayout">
          <item>
           <widget class="QPushButton" name="refreshItemButton">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="borrowButton">
            <property name="text">
             <string>Borrow Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="itemTable">
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Item Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="borrowTab">
       <attribute name="title">
        <string>My Borrowed Items</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="borrowButtonLayout">
          <item>
           <widget class="QPushButton" name="refreshBorrowButton">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="returnButton">
            <property name="text">
             <string>Return Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="borrowTable">
          <column>
           <property name="text">
            <string>Item ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Item Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Borrow Date</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adminTab">
       <attribute name="title">
        <string>Admin Panel</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="adminLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 14pt; font-weight: bold; color: #2c3e50;</string>
          </property>
          <property name="text">
           <string>ADMIN PANEL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="adminButtonGrid">
          <property name="spacing">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="addItemButton">
            <property name="styleSheet">
             <string notr="true">padding: 15px; font-size: 12pt; font-weight: bold; border: none; border-radius: 8px; min-width: 200px; min-height: 60px; background-color: #27ae60; color: white;</string>
            </property>
            <property name="text">
             <string>Add New Item</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="editItemButton">
            <property name="styleSheet">
             <string notr="true">padding: 15px; font-size: 12pt; font-weight: bold; border: none; border-radius: 8px; min-width: 200px; min-height: 60px; background-color: #f39c12; color: white;</string>
            </property>
            <property name="text">
             <string>Edit Item</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="deleteItemButton">
            <property name="styleSheet">
             <string notr="true">padding: 15px; font-size: 12pt; font-weight: bold; border: none; border-radius: 8px; min-width: 200px; min-height: 60px; background-color: #e74c3c; color: white;</string>
            </property>
            <property name="text">
             <string>Delete Item</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="viewUsersButton">
            <property name="styleSheet">
             <string notr="true">padding: 15px; font-size: 12pt; font-weight: bold; border: none; border-radius: 8px; min-width: 200px; min-height: 60px; background-color: #3498db; color: white;</string>
            </property>
            <property name="text">
             <string>View All Users</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="viewBorrowsButton">
            <property name="styleSheet">
             <string notr="true">padding: 15px; font-size: 12pt; font-weight: bold; border: none; border-radius: 8px; min-width: 200px; min-height: 60px; background-color: #9b59b6; color: white;</string>
            </property>
            <property name="text">
             <string>View All Borrows</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="validateReturnButton">
            <property name="styleSheet">
             <string notr="true">padding: 15px; font-size: 12pt; font-weight: bold; border: none; border-radius: 8px; min-width: 200px; min-height: 60px; background-color: #1abc9c; color: white;</string>
            </property>
            <property name="text">
             <string>Validate Returns</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
